cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -319,6 +333,17 @@ public ConfigMap generateMetricsConfigMap() {
         }
     }
 
+    /**
+     * Generates a log ConfigMap according to configured defaults
+     * @return The generated ConfigMap
+     */
+    public ConfigMap generateLogConfigMap(Vertx vertx) {
+        this.vertx = vertx;
+        Map<String, String> data = new HashMap<>();
+        data.put(LOG_CONFIG_FILE, getLogConfig());
```

Comment:
As we discussed earlier with @tombentley, using ConfigMap makes  it harder to trigger rolling update in case of change. Is the logging configuration config map somehow plugged into the `Diff` mechanism so that rolling update is triggered when the config map is updated?

Reply:
Yes. When the config map is changed, new log levels are set up. It happens due to awesome design made by Paolo I believe!